elastic/datadog_checks/elastic/elastic.py

```diff
@@ -40,6 +41,10 @@ def __init__(self, name, init_config, instances):
         # Host status needs to persist across all checks
         self.cluster_status = {}
 
+        if self.instance.get('auth_type') == 'aws' and self.instance.get('url'):
+            self.HTTP_CONFIG_REMAPPER = self.HTTP_CONFIG_REMAPPER.copy()
```

Comment:
Why do you need to copy?

Reply:
To break the reference to the global one, otherwise a mutation would affect all instances